std_msgs/Header header
float64 latitude
float64 longitude
float64 altitude
float64 utm_easting
float64 utm_northing
uint8 zone
string letter
float64 hdop
string gngga_read
int32 fix_quality
time current_time
